syntax = "proto3";

option csharp_namespace = "Generated";

message KeyPressResponse {
	string content = 1;
	bool result = 2;
}

message KeyPressRequest {
	string content = 1;
}

service RemoteKeyboardService {
	rpc SendKeyboardCalls(stream KeyPressRequest) returns (stream KeyPressResponse);
}